malleefowl/custom_workflow.py

```diff
@@ -0,0 +1,569 @@
+"""
+Run custom workflow without prior knowledge of the underlying component except the fact that they are WPS
+The workflow must have the following structure:
+Dict of 2 elements :
+  - name : Workflow name
```

Comment:
With small changes this could be valid Sphinx documentation. e.g. : 

:name: blah
:blah2: "blah2"

...